.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
```

Comment:
Which plugin do you use for indent detection?

I've seen both modeline, editorconfig, vim-sleuth, as well as recent lua alternatives.

Reply:
Just using the (now) build-in editorconfig support + https://github.com/nmac427/guess-indent.nvim